lib/bibliothecary/parsers/pypi.rb

```diff
@@ -179,20 +179,51 @@ def self.parse_setup_py(file_contents, options: {})
         deps
       end
 
+      NoEggSpecified = Class.new(ArgumentError)
```

Comment:
What is egg?

Reply:
An "egg" is the name of an older format for distributing Python code, nowadays mostly replaced by "wheel"s. However, PyPI uses `egg` in the fragment of the URL to identify which package in the PyPI index the git repo represents.

https://stackoverflow.com/a/55526482